tripy/examples/nanogpt/model.py

```diff
@@ -76,19 +75,15 @@ def __init__(self, config):
         self.zeros = tp.zeros((1, 1, self.seq_len, self.seq_len), dtype=config.dtype)
 
     def __call__(self, x: tp.Tensor):
-        B, T, C = tp.reshape(x.shape[0], (1,)), tp.reshape(x.shape[1], (1,)), tp.reshape(x.shape[2], (1,))
+        B, T = x.shape[0], x.shape[1]
```

Comment:
```suggestion
        B, T = x.shape[0:2]
```

Reply:
Why do we call it `T` here? Isn't it sequence length? Usually that's `S`.

Reply:
Yes it is sequence length, I think it's pretty common to use `T`, nanoGPT uses T: https://github.com/karpathy/nanoGPT/blob/master/model.py#L53